fsd_utils/healthchecks/healthcheck.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+
+class Healthcheck(object):
+
+    def __init__(self, app):
+        self.flask_app = app
+        self.flask_app.add_url_rule("/healthcheck", view_func=self.healthcheck_view)
+        self.checkers = []
+
+    def healthcheck_view(self):
+        responseCode = 200
+        response = {"checks":[]}
+        for checker in self.checkers:
+            try:
+                result = checker.check()
+                current_app.logger.debug(f"Check {checker.name} returned {result}")
```

Comment:
If current_app doesn't exist at this point will this view result in a 500 (without any descriptive json) when we run the FlaskRunningChecker? i.e. the check() will run correctly and return but this view won't return because of the current_app.logger? Could we just use the default logger to avoid this?